### DISCUSSION TITLE
Assets imported with history Checkout button greyed out

### DISCUSSION THREAD

**Discussion by raviax** [snipe/snipe-it#12130] [Q&A]
![image](https://user-images.githubusercontent.com/7872640/202206562-10bf5390-8672-4617-8ed9-419c3c80f299.png)

Hello, I've imported a list of assets that were checked out through the history import. These assets show up as Deployable, but the Checkout button is greyed out. If I open the asset and select Check In from the Actions menu, I get an error saying it's already checked in, but I still cannot check it out to the person listed in the History. I've tried changing the status from Deployable and back to Deployable, but I still can't check it out or assign it to the user. How do I fix this? I'd like them to be "checked out" to the user they are assigned to in the history.

![image](https://user-images.githubusercontent.com/7872640/202208271-4ca7f228-8fc2-474e-b909-1806683787ac.png)

**Comment by zbrand55** [comment]
I am just throwing out guesses, but maybe there is some required information missing from the asset or the user that is causing the grey-out.  When hovering over the button, do you get a popup message? My example (pic attached) is from a Pending Asset (I can see yours is set as deployable).

![snipecheckout](https://user-images.githubusercontent.com/104520984/203599666-cc04c4d4-01ba-4fc9-bba4-53d512a7a6be.png)

**Reply by raviax** [reply]
Yes, mine says the same thing. When I edit the asset, all required fields are filled out. When I look at the user that the history has it checked out to, it does not show up under their assets.
